### PR TITLE
Views

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:
- Linked Model and Controller methods.
- Added a message to the default layout to display in all views.
- Made views display the correct information by refactoring the views files.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 <h1>Users</h1>
 <p>You will be able to see the users listed here.</p>
+<ul>
+  <% @users.each do |user| %>
+    <li>
+      <h2><%= user.name %></h2>
+      <p>Number of posts: <%= user.posts_counter %></p>
+    </li>
+  <% end %>
+</ul>
```
Comment (oxenprogrammer) [inline app/views/users/index.html.erb]: **Changes Required**
- [x] Clicking on each user takes you to their show page. Let us please add this link

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @rpire, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (rpire) [reply app/views/users/index.html.erb]: Changes applied.

**Comment by rpire** [comment]
Hello, @oxenprogrammer 👋

Thank you for reviewing my work! You are right. Adding links makes navigation through the app a lot easier. I didn't think of adding them before because I didn't see them in the requirements, but I'm glad you have brought them up. 🤓

Thanks again for the review. The required changes are complete and, I will proceed to request another review. 🔎

Cheers and Happy coding to you too! 🐱‍💻

**Review by oloomoses — APPROVED** [review]
Hi @rpire, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
Hello, @oloomoses 🙌

Thank you for reviewing my work. I will proceed to merge these changes into the `dev` branch 🚀

Cheers and Happy coding to you too. 🐱‍💻

**Pull request merged**